Submit DEI Standing Committee Charter (#160)
jupyter/governance: charters/DeiCharter.md

# Diversity Equity and Inclusion Committee Charter

Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.

The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council (EC) and the Software Steering Council (SSC) for approval.

## Purpose 

> The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 

Reviewer: ```suggestion
The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, contributors, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. The DEI group impacts all Jupyter forums, including but not limited to software, working groups, standing committees, JupyterCon, social media, GitHub, the EC, and the SSC.
```

(Including @afshin 's suggestion and addressing @ellisonbg and @fperez 's request)

## Mission/Areas of Responsibility

> The DEI standing committee is responsible for: 

Reviewer 2: ```suggestion
The DEI standing committee is responsible to: 
```

Reviewer 3: I would suggest instead (it sounds more readable to me, but I'll defer to a native speaker) `The responsibilities of the DEI standing committee are to:`

* Propose, execute, and evaluate the impact of DEI initiatives
* Research, establish, and collect metrics related to DEI
* Advocate for increased participation of underrepresented groups in the Jupyter community. 
* Serve as a point of contact for DEI topics for the global Jupyter community
* Include the topic of DEI in discussions and spaces within Project Jupyter.
* Recommend and promote training, educational, and motivational materials on DEI 
* Flag DEI issues or potential issues to the EC or the SSC

## Meetings
* Frequency - this committee should meet at least once every two weeks.
* The Jupyter Decision-Making Guide will be used to make official decisions. 

> * To ensure active participation, a member who is unable to join for more than two thirds of regularly scheduled weekly meetings in a 90 day period may be asked to step down from the committee.

Reviewer 4: Thx @Zsailer for asking about the sync/async as timezone. Sorry to hijack this PR, but would like to mention we had the exact same discussions during the last Software Steering Council (SSC). Should those aspects be defined for all groups, or are they a delegated to the groups? In the latter case the groups would be free to decide and evolve the rules by themselves and those aspects should not be mentioned in the charter IMHO.

Thinking loud, I would prefer the governance to define some guidance for all groups to have a basis and common framework to avoid discussion and too much disparity across groups.

BTW the timezone question is a perfect topic for the DEI group to ensure enough on-call representation from around the globe :)

> * To ensure active participation, a member who is unable to join for more than two thirds of regularly scheduled weekly meetings in a 90 day period may be asked to step down from the committee.

Reviewer 5: The delay linked to vote mentioned in the Decision-Making Guide is a year. Is there a specific reason to have 90 days here?

> Voter participation and quorum. All members of a council are required to participate in at least 2/3 of formal votes of that council per calendar year (councils can decide how to account for the specifics of this in low-activity projects, etc.). Members that have not met the 2/3 vote participation threshold for a year will automatically be asked to step down at the end of that year.

> * To ensure active participation, a member who is unable to join for more than two thirds of regularly scheduled weekly meetings in a 90 day period may be asked to step down from the committee.

Reviewer: @Zsailer Thanks for bringing this up. The "time zone problem", as I'll call it for short, affects our team and many others. On the DEI committee, we have made an effort to set weekly meetings at a time that is convenient for most of our members. We currently have four members on the DEI committee, all of whom are in the Americas, and we are focused on synchronous participation. We are definitely open to holding our meetings at different times if we find someone for whom our current meeting time is not suitable. We use the word "may" in our charter to say that it is not automatic to exclude a person for being unable to attend 2/3 of the meetings. Other committees should set expectations for themselves. Does this address your concern?

@fcollonval Our charter describes an expectation (but not a requirement) to attend more than 2/3 of scheduled weekly meetings. The decision-making guide describes a requirement to participate in at least 2/3 of formal votes. Because a scheduled weekly meeting is not as formal, we don't consider our guidance to be in conflict with the decision-making guide.

## Committee Reports

> * Reports will be submitted to the EC half-yearly on or before the 15th of July and January for the preceding calendar half.

Reviewer 6: Just clarifying, is the intention that these reports are for the Executive Council only?

Reviewer 7: Yes, that's correct. I do think it's fair to ask this committee to present some form of public communication whether it be in the form of a blog post, a YouTube video, or office hour. I will add this to our next agenda for discussion. Thanks for bringing this up.

> * Reports should include the following metrics:

Reviewer: ```suggestion
* Reports are encouraged to include metrics that the Committee establishes are relevant, informative and help in decision making, such as:
```

# Diversity Equity and Inclusion Committee Charter

Project Jupyter welcomes and encourages participation in our community by people of all backgrounds and identities. We are committed to promoting and sustaining a culture that values mutual respect, tolerance, and learning. We work together as a community to help each other live out these values.

> The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.

Reviewer 8: Because the DEI group is a standing committee, change to this charter need to be approved by both the EC and SSC.

> The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.

Reviewer 9: As a Standing Committee, I think the EC and SSC both need to approve?

## WHY: Purpose 

Why is the committee being created? How does this group contribute to Jupyter? 

> The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 

Reviewer 8: Should we include details about the scope of the DEI group, namely that it covers all contexts of Jupyter work: software, working groups, standing committees, JupyterCon, online forums, Github, EC, SSC, etc.?

Reviewer 3: Agreed!

> The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 

Reviewer 9: Maybe add "contributors" to the scope?
```suggestion
The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, contributors, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
```

> The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 

Reviewer 10: ```suggestion
The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, contributors, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 
```

via @fperez

The DEI standing committee is responsible for: 
* Propose, execute, and evaluate the impact of DEI initiatives
* Research, establish, and collect metrics related to DEI
* Advocate for increased participation of underrepresented groups in the Jupyter community. 
* Serve as a point of contact for DEI topics for the global Jupyter community
* Include the topic of DEI in discussions and spaces within Project Jupyter.
* Recommend and promote training, educational, and motivational materials on DEI 
* Flag DEI issues or potential issues to the Executive Council (EC) or the Software Steering Council (SSC)

## HOW: Meetings
* Frequency - this committee should meet at least once every two weeks.
* The Jupyter Decision-Making Guide will be used to make official decisions. 
* To ensure active participation, a member who is unable to join for more than two thirds of regularly scheduled weekly meetings in a 90 day period may be asked to step down from the committee.

## HOW: Committee Reports
* Reports will be submitted to the Executive Council half-yearly on or before the 15th of July and January for the preceding calendar half.

> * Reports should include the following metrics:

Reviewer 3: I would replace with "Reports are encouraged to include metrics that the Committee establishes are relevant, informative and help in decision making" or similar, rather than hard-coding any specific metric in this charter.

The Diversity Equity & Inclusion (DEI) Committee is a standing committee established under Jupyter’s governance model. The committee reviews its charter annually and may make changes at any time during the year. Substantive changes to the purpose or mission must be submitted to the Executive Council for approval.

## WHY: Purpose 

Why is the committee being created? How does this group contribute to Jupyter? 

The purpose of this committee is to lead strategic initiatives designed to improve the diversity, equity, and inclusion of Project Jupyter community leaders, core developers, and end users across the globe. For the long-term sustainability of our community, we champion efforts to build an open source environment where people feel respected, valued, and encouraged to continue participating. 

## WHAT: Mission/Areas of Responsibility

What will this committee work on? How will the purpose described above be accomplished?

> The DEI standing committee is responsible to: 

Reviewer 3: "responsible for"?

> * Propose, execute, and evaluate the impact of DEI initiatives

Reviewer: ```suggestion
* Propose, execute, collect metrics, and evaluate the impact of DEI initiatives
```

* Serve as a point of contact for DEI topics for the global Jupyter community
* Serve as DEI ambassadors by including this topic in the discussions and spaces within Project Jupyter
* Recommend and promote training, educational, and motivational materials on DEI 
* Flag DEI issues or potential issues to the Executive Council (EC) or the Software Steering Council (SSC)

## WHO: Committee Composition
### Members
Members should affirm the opening paragraph of this charter and agree to the purpose of this committee

> Committees should consist of at least 2 members serving a 1 year term with an option for unlimited renewal 

Reviewer 11: `Committees` (plural) is used in a couple of places in this doc- are there multiple (sub?)committees?

> Committees should consist of at least 2 members serving a 1 year term with an option for unlimited renewal 

Reviewer: ```suggestion
The committee should consist of at least 2 members serving a 1 year term with an option for unlimited renewal 
```

### General Members
Responsibilities:
* Attend meetings as scheduled by the vice president/co-chair(s)
* Complete onboarding tasks in a timely manner
* Assist in carrying out the committee’s objectives
* Adding new members
* An existing member may nominate a new member for the committee
* Nominations are discussed at committee meetings;  nominations are accepted by a simple majority 
* In extraordinary circumstances, the Chair may reject the nomination
### Leadership Members:
For smaller committees (8 or less), only a chair is required. For large committees (9 or more), three leadership positions are required. One person may hold a leadership position and also serve as an SSC rep.
#### Chair
* Board appointed or elected by committee members for a one-year term with an option to serve an additional one-year term upon member approval.
* May be submitted to the Executive Council for approval as part of a new committee proposal (one-year term)
* Responsibilities:
  * Calls for scheduled meetings
  * Sets the agenda for meetings

>   * Enforces Jupyter’s Code of Conduct as needed

Reviewer 12: I would remove this CoC bullet here and under vice-chair if Jupyter is going to have a Code of Conduct standing committee so it is clear who is responsible for Code of Conduct issues and remediation.

Perhaps change to something along the lines of:
- Provide feedback and insights to the Code of Conduct committee

Author: We actually were already planning to remove the "Who" section - just pushed the update without that section

#### Vice-Chair
* Board appointed or elected by committee members for a one-year term with an option to serve an additional one-year term upon member approval.
* May be submitted to the Executive Council for approval as part of a new committee proposal (one-year term)
* Responsibilities:
  * Partners with the chair to host productive meetings
  * Fill in for the chair as needed to lead meetings
  * Enforces Jupyter’s Code of Conduct as needed
#### Secretary
* Selected by the Executive Council or elected by committee members for a one-year term with an option to serve an additional one-year term upon member approval.
* Responsibilities:
  * Records all meeting minutes
  * Submits reports to the Executive Council

## HOW: Meetings

> * Frequency - established by the committee but it is suggested that committees meet at least weekly or bi-monthly. 

Reviewer: ```suggestion
* Frequency - this committee should meet at least once every two weeks.
```
